src/prisma/schedule.ts

```diff
@@ -37,3 +39,108 @@ async function initiateSchedule(scheduleyear: number) {
   });
   return schedule;
 }
+
+async function updateCurrentSchedule(
+  scheduleId: any,
```

Comment:
Frontend will either return a `string | undefined` here

Reply:
`Error: src/resolvers/index.ts(252,42): error TS2345: Argument of type 'InputMaybe<string> | undefined' is not assignable to parameter of type 'string | undefined'.
  Type 'null' is not assignable to type 'string | undefined'.
Error: Process completed with exit code 2.`

I'm getting these errors if I use string | undefined

Reply:
I'm just going to revert this back to any so we can just push the changes